### DISCUSSION TITLE
Print QSL Queue

### DISCUSSION THREAD

**Discussion by M0TZO** [magicbug/Cloudlog#1021] [Ideas / Feature Requests]
Been adding some QSLs to the Print queue a likely going to use an online QSL service.

Wondering if could have some info (without downloading ADIF) on the Print page.

- Number of QSLs in the Queue (& maybe how many callsigns)
- List of callsigns, so can edit list (eg remove, or add other QSOs from same callsign
- _I'm sure I might think of some more_

**Comment by dg9vh** [comment]
Within the print list are all qsos marked as requested or queued for qsl
send in the qsl tab.

73 de Kim
DG9VH

Paul GIBSON ***@***.***> schrieb am Do., 6. Mai 2021, 22:07:

**Reply by M0TZO** [reply]
All are marked for print with `<QSL_SENT:1>Q` as Queued, at least how I was doing it.

Hope this helps, and want you wanted.

**Comment by dg9vh** [comment]
And R

Paul GIBSON ***@***.***> schrieb am Fr., 7. Mai 2021, 00:27:

**Comment by AndreasK79** [comment]
I see that view could use some more information. I'll have a look at it.

**Comment by AndreasK79** [comment]
Check out the latest code base for some improvements here.

**Comment by AndreasK79** [comment]
@M0TZO I added some further enhancements now with PR #1112 . In the list, you have a new button that says "Open QSO list". You will then get a list with QSOs for this callsign. You can now add these to the queue if you want.
